examples/src/main/scala/org/apache/spark/examples/mllib/MovieLensALS.scala

```diff
@@ -18,14 +18,14 @@
 package org.apache.spark.examples.mllib
 
 import scala.collection.mutable
-
 import org.apache.log4j.{Level, Logger}
 import scopt.OptionParser
-
 import org.apache.spark.{SparkConf, SparkContext}
 import org.apache.spark.SparkContext._
 import org.apache.spark.mllib.recommendation.{ALS, MatrixFactorizationModel, Rating}
 import org.apache.spark.rdd.RDD
+import org.apache.spark.mllib.evaluation.RankingMetrics
+import org.jblas.DoubleMatrix
```

Comment:
organize imports (https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports).

Do not use JBLAS because we plan to remove it in 1.4. Use breeze/netlib-java instead.